pandas-stubs/io/formats/style.pyi

```diff
@@ -1,97 +1,267 @@
+        axis: Axis = ...,
+        level: Level | list[Level] | None = ...,
+        names: bool = ...,
+    ) -> Styler: ...
     def background_gradient(
         self,
-        cmap=...,
-        low=...,
-        high=...,
-        axis=...,
-        subset=...,
-        text_color_threshold=...,
+        cmap: str = ...,
+        low: float = ...,
+        high: float = ...,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        text_color_threshold: float = ...,
         vmin: float | None = ...,
         vmax: float | None = ...,
-    ): ...
-    def set_properties(self, subset=..., **kwargs): ...
+        gmap: Sequence | None = ...,
+    ) -> Styler: ...
+    def text_gradient(
+        self,
+        cmap: str = ...,
+        low: float = ...,
+        high: float = ...,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        vmin: float | None = ...,
+        vmax: float | None = ...,
+        gmap: Sequence | None = ...,
+    ) -> Styler: ...
+    def set_properties(self, subset: Subset | None = ..., **kwargs) -> Styler: ...
     def bar(
         self,
-        subset=...,
-        axis: int = ...,
+        subset: Subset | None = ...,
+        axis: Axis | None = ...,
+        *,
+        color: str | list | tuple | None = ...,
+        cmap: Any | None = ...,
+        width: float = ...,
+        height: float = ...,
+        align: str | float | Callable = ...,
+        vmin: float | None = ...,
+        vmax: float | None = ...,
+        props: str = ...,
+    ) -> Styler: ...
+    def highlight_null(
+        self,
+        null_color: str = ...,
+        subset: Subset | None = ...,
+        props: str | None = ...,
+    ) -> Styler: ...
+    def highlight_max(
+        self,
+        subset: Subset | None = ...,
         color: str = ...,
-        width: int = ...,
-        align: str = ...,
-        vmin=...,
-        vmax=...,
-    ): ...
-    def highlight_max(self, subset=..., color: str = ..., axis: int = ...): ...
-    def highlight_min(self, subset=..., color: str = ..., axis: int = ...): ...
+        axis: Axis | None = ...,
+        props: str | None = ...,
+    ) -> Styler: ...
+    def highlight_min(
+        self,
+        subset: Subset | None = ...,
+        color: str = ...,
+        axis: Axis | None = ...,
+        props: str | None = ...,
+    ) -> Styler: ...
+    def highlight_between(
+        self,
+        subset: Subset | None = ...,
+        color: str = ...,
+        axis: Axis | None = ...,
+        left: Scalar | Sequence | None = ...,
+        right: Scalar | Sequence | None = ...,
+        inclusive: str = ...,
+        props: str | None = ...,
+    ) -> Styler: ...
+    def highlight_quantile(
+        self,
+        subset: Subset | None = ...,
+        color: str = ...,
+        axis: Axis | None = ...,
+        q_left: float = ...,
+        q_right: float = ...,
+        interpolation: str = ...,
+        inclusive: str = ...,
```

Comment:
`Literal[‘both’, ‘neither’, ‘left’, ‘right’]` although I think we have a type for that in `_typing.pyi`

Reply:
Fixed.